lib/network-layer/src/Cardano/Wallet/Network/Implementation.hs

```diff
@@ -430,7 +429,7 @@ withNetworkLayer tr pipeliningStrategy np conn ver tol action = do
 
 -- | Network parameters and protocol parameters for the node's current tip.
 data NetworkParams = NetworkParams
-    { protocolParams :: MaybeInRecentEra Write.PParams
+    { protocolParams :: Read.EraValue Read.PParams
     , protocolParamsLegacy :: ProtocolParameters
```

Comment:
So now this `protocolParamsLegacy` could be dropped — if we wanted to

Reply:
Yes. The question is whether this investment into the legacy wallet is worth the effort. At the moment, I'd like to freeze it in place, at least.